app/views/profiles/show.html.erb

```diff
@@ -2,15 +2,77 @@
 <%= render "partials/welcome_modal" %>
 
 <div class="content-wrapper-narrow content-container">
+  <!--
   <% if @profile.supported? %>
     <%= t(".supported", zipcode: @profile.zipcode) %>
   <% else %>
     <%= t(".unsupported", zipcode: @profile.zipcode) %>
   <% end %>
+  -->
 
-  <%= render("locations/form", location: @profile.location || @profile.build_location) %>
+  <header class="profile-header">
+    <h2 class="profile-header__heading"><span>firstName</span> <span>lastName</span></h2>
```

Comment:
Is it important for these to be wrapped in `span`s? I don't see any styles for it